.github/labeler.yml

```diff
@@ -0,0 +1,87 @@
+api:
+  - "app/bundles/ApiBundle/**"
+  
+assets:
+  - "app/bundles/AssetBundle/**"
+  
+campaigns:
+  - "app/bundles/CampaignBundle/**"
+  
+categories:
+  - "app/bundles/CategoryBundle/**"
+  
+channels:
+  - "app/bundles/ChannelBundle/**"
+  
+companies:
+  - "**/*company*"
+  
+configuration:
+  - "app/bundles/ConfigBundle/**"
+  
+dashboard:
+  - "app/bundles/DashboardBundle/**"
+  
+dynamic content:
+  - "app/bundles/DynamicContentBundle/**"
+  
+email:
+  - "app/bundles/EmailBundle/**"
+  
+forms:
+  - "app/bundles/FormBundle/**"
+  
+import-export:
+  - "**/*import*"
+  
+installation:
+  - "app/bundles/InstallBundle/**"
+  
+integrations:
+  - "app/bundles/IntegrationsBundle/**"
+  
+landing pages:
+  - "app/bundles/PageBundle/**"
+  
+marketplace:
+  - "app/bundles/MarketplaceBundle/**"
+  
+notifications:
+  - "app/bundles/NotificationBundle/**"
+  
+plugin:
+  - "app/bundles/PluginBundle/**"
+  
+points-scoring:
+  - "app/bundles/PointBundle/**"
+  
+queue:
+  - "app/bundles/QueueBundle/**"
```

Comment:
The queue bundle will be removed in M6 and replaced by Symfony's way how to do queues. So perhaps we don't need it here as it won't get any adjustments anymore.

Reply:
Removed